packages/next/src/server/image-optimizer.ts

```diff
@@ -44,23 +39,26 @@ const VECTOR_TYPES = [SVG]
 const BLUR_IMG_SIZE = 8 // should match `next-image-loader`
 const BLUR_QUALITY = 70 // should match `next-image-loader`
 
-let sharp: typeof import('sharp') | undefined
+let sharp: typeof import('sharp')
```

Comment:
How does TypeScript handle when `sharp` is not installed? Will this just quietly be `undefined`?

Reply:
Previously, the `catch` would consume the error so there was a scenario where sharp is undefined.

That is no longer the case as you can see, `catch` always rethrows so there is never a case when sharp is undefined.